npf/types/web/web.py

```diff
@@ -0,0 +1,30 @@
+import uuid
+
+from npf_web_extension import app
+
+def prepare_web_export(datasets, all_results_df, path):
+
+    # Getting parameters and measurements
+    name = "undefined"
+    parameters = []
+    measurements = []
+    for testie, _, runs in datasets:
+      name = testie.get_title()
+      for run, results in runs.items():
+        measurements = list(results.keys())
+        parameters = list(run.variables.keys())
+        break
+      break
+
+    # Preparing configuration data
+    configurationData = {

```

Comment:
Somehow the variable names from test.config["var_names"] should be used to have the right axis names

Reply:
Well, I would do that but it seems that var_names are hardcoded by default (see https://github.com/dpcodebiz/npf/blob/c7e6dc6374d775875027e0d1f7244b515ec05f25/npf/section.py#L690 ). Hence, if I use it, for tests like examples/math.npf, graphs axis would be "LATENCY" which has no sense imo. Thus, should we rather remove the default value first then proceed or just leave it the way it is?

Reply:
No that's not how it works. The idea for "RESULT-NAME"  is to look for result in the "var_names" dictionnary. Some default names like Latency are there, but the test file can use var_names+={LOG:Logarithm} for math.npf for instance.

Reply:
Okay, understood. I just added it, it should be good now!